main/Makefile

```diff
@@ -133,8 +133,14 @@ install:
 	done
 	$(INSTALL) -m 755 $(BINDIR)/mimic_time $(DESTDIR)$(INSTALLBINDIR)
 #       The libraries: static and shared (if built)
+	# original cp flags for unix/linux systems
 	cp -pd $(mimic_LIBS_deps) $(DESTDIR)$(INSTALLLIBDIR)
```

Comment:
Can the copy flags be included in the configure step?

something along the lines of
```
if test "$host_os" = "osx" ; then                                               
   COPYFLAGS="-pR"                                                             
else                                                                            
   COPYFLAGS="-pd"                                                       
fi
[...]
AC_SUBST(COPYFLAGS)
```
in config/config.in
```
COPYFLAGS = @COPYFLAGS@
```

and in main/Makefile replace the copy flags with the variable.
```
cp $(COPYFLAGS) [...]
```

I'm not sure what mac osX $host_os will be set to but you can check it by inserting an
echo $host_os in the configure.in.

(Note that you need *autoconf* tool to update the configure script from configure.in)